app/models/host.rb

```diff
@@ -1568,6 +1524,30 @@ def scan_from_queue(taskid=nil)
     _log.info("Scanning #{log_target}...Complete - Timings: #{t.inspect}")
   end
 
+  def scan_refreshes(log_target, task)
+    # Firewall Rules and Advanced Settings go through EMS so we don't need Host credentials
+    scan_refresh("Firewall Rules", log_target, task) { refresh_firewall_rules }
+    scan_refresh("Advanced Settings", log_target, task) { refresh_advanced_settings }
+    scan_refresh("IPMI information", log_target, task) { refresh_ipmi } unless ext_management_system
```

Comment:
I think the conditional's getting lost here; consider using a block `unless` instead?